darts/models/kalman.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+import pandas as pd
+
+from abc import ABC
+
+from typing import Optional
+from filterpy.kalman import KalmanFilter
+
+from .smoothing_model import SmoothingModel
+from ..timeseries import TimeSeries
+
+
+class Kalman(SmoothingModel, ABC):
+
+    def __init__(
+            self, 
+            dim_x: int = 1, 
+            dim_z: int = 1, 
+            x = np.array([[1.]]),
+            F = np.array([[1]]),
+            H = np.array([[1]]),
+            P = np.array([[1000.,    0.], [   0., 1000.] ]),
+            Q = 0.1,
+            kf: Optional[KalmanFilter] = None
+        ):
+        """ Kalman filter model
+        This model doesn't predict the future values, instead it predicts 
+        more precise values compared to noise interfered data
+        
+        Parameters
+        ----------
+        dim_x : int
+            Size of the Kalman state vector
+        dim_y : int
+            Size of the measurement vector
+        x : numpy array
+            filter state estimate
+        F : numpy array
```

Comment:
It would be nice to state what are the expected dimensions of these matrices (e.g. `dim_x x dim_x`)